cmd/zoekt-webserver/main.go

```diff
@@ -481,8 +482,16 @@ func (s *loggedSearcher) log(ctx context.Context, q query.Q, opts *zoekt.SearchO
 			sglog.Duration("opts.MaxWallTime", opts.MaxWallTime),
 			sglog.Int("opts.MaxDocDisplayCount", opts.MaxDocDisplayCount),
 		)
+
 	if err != nil {
-		logger.Error("search failed", sglog.Error(err))
+		switch {
+		case errors.Is(err, context.Canceled):
+			logger.Error("search canceled", sglog.Error(err))
+		case errors.Is(err, context.DeadlineExceeded):
+			logger.Error("search timeout", sglog.Error(err))
```

Comment:
these feel like warnings or even lower TBH

Reply:
Tend to agree actually